modules/settings/assets/js/components/sidebar-app-bar/index.js

```diff
@@ -0,0 +1,34 @@
+import AppBar from '@elementor/ui/AppBar';
+import Box from '@elementor/ui/Box';
+import IconButton from '@elementor/ui/IconButton';
+import Toolbar from '@elementor/ui/Toolbar';
+import Typography from '@elementor/ui/Typography';
+import { __ } from '@wordpress/i18n';
+import { useSettings } from '../../hooks';
+import {
+	ElementorLogo,
+	SquareRoundedChevronsLeft,
+} from '../../icons';
+
+export const SidebarAppBar = () => {
```

Comment:
Would that be clear enough to just keep it as `Sidebar`? The current naming sounds a bit mouthful.